test/smoke/src/main.ts

```diff
@@ -179,7 +178,7 @@ function parseQuality(): Quality {
 //
 if (!opts.web) {
 	let testCodePath = opts.build;
-	let electronPath: string;
+	let electronPath: string | undefined;
```

Comment:
This change seems to be unrelated to removal of mkdirp
```suggestion
	let electronPath: string;
```

Reply:
thats ok 👍